### PR TITLE
Listen for changes inside WebView

### PR DESCRIPTION
**Pull request by mqtik**

### CODE REVIEW

**Comment by mqtik** [comment]
That should do it.

**Comment by mqtik** [comment]
@imnapo care to merge it?

File: src/CNEditor.js [hunk 1]
```diff
@@ -205,7 +209,13 @@ export default class CNEditor extends Component {
             this.webViewRef.postMessage(jsonString);
         } 
     }
-
+    
+    onValueChanged = (data) => {
+        this.props.initialHtml = data;
```
Comment (imnapo) [inline src/CNEditor.js]: it is not valid to change props. so I will remove this line.
also we use 'initialHtml' just once on load , so changing it has no effect.

**Comment by imnapo** [comment]
@mqtik  Excellent job,

**Pull request merged**